docs/user-guide/how_to_enable_dra.md

```diff
@@ -0,0 +1,76 @@
+# How to Enable Dynamic Resource Allocation (DRA) in Volcano Scheduler
```

Comment:
This file should also be added to website after merged.

Reply:
OK